### ISSUE TITLE
Minimum pooling operation. 

### ISSUE THREAD

**Issue by Santosh-Gupta** [tensorflow/tensorflow#40491]
**System information**
- TensorFlow version (you are using): Latest as of 6/15/20
- Are you willing to contribute it (Yes/No): Yes.

**Describe the feature and the current behavior/state.**

Tensorflow/Keras has max pooling, and average pooling, but there is no minimum pooling. A minimum pooling operation feature would be great to have. 

**Will this change the current api? How?**

There could be a tf.nn.min_pool

**Who will benefit with this feature?**

Minimum pooling of embeddings can be powerful features, especially if those embeddings that are created with architectures which use a tanh activation (instead of rulu, sigmoid, etc.), since these embeddings would contain useful information in both the positive and negative direction of the embedding.

**Comment by Santosh-Gupta** [comment]
nevermind, it looks like https://www.tensorflow.org/api_docs/python/tf/math/reduce_min does the job
